enterprise/frontend/src/embedding-sdk/cli/utils/ask-for-db-connection-info.ts

```diff
@@ -107,27 +116,44 @@ const getIntegerFieldDefault = (field: EngineField, engine: string) => {
 };
 
 const askSectionChoice = async (field: EngineField) => {
-  // Snowflake allows to connect with either hostname or account name.
+  // Postgres allows connecting with either password or an authentication provider.
+  if (field.name === "use-auth-provider") {
+    const choice = await select({
+      message:
+        "Do you want to connect with password or an authentication provider?",
+      choices: [
+        { name: "Password", value: "password" },
+        { name: "Auth Provider", value: "auth-provider" },
+      ],
+      default: "password",
+    });
+
+    return choice === "auth-provider";
+  }
+
+  // Snowflake allows connecting with either hostname or account name.
   if (field.name === "use-hostname") {
     const choice = await select({
       message: "Do you want to connect with hostname or account name?",
       choices: [
         { name: "Hostname", value: "hostname" },
         { name: "Account name", value: "account" },
       ],
+      default: "hostname",
```

Comment:
![image](https://github.com/user-attachments/assets/9d70cedb-6cf6-4c98-a5c4-492c1cd9fc1f)
From the actual default value I got, I assume `false` means it's not using hostname, so `account` should be the default value.

Also, I noticed that all `section` type fields are actually booleans. I'd want to see the values in the choices being `true` or `false` instead for clarity.

Also, you could probably use `field.default` for this if you set value as a boolean.

Reply:
@WiNloSt Good call! Will update this to match the field's representation 👍🏻

Reply:
[Fixed in this commit.](https://github.com/metabase/metabase/pull/46810/commits/d050b1d9ff6da03ccf68be8e3624f7e776ce08d5)